src/extension.ts

```diff
@@ -127,6 +138,73 @@ export function activate(context: ExtensionContext) {
         languageClient.sendRequest(ExecuteCommandRequest.type, request);
     }));
 
+    // Create the "open hub" command
+    // This is activated from the status bar (see below)
+    context.subscriptions.push(
+    commands.registerCommand("sourcery.hub.open", () => {
+
+      // Command is handled by the language server
+      languageClient
+        .sendRequest(ExecuteCommandRequest.type, {
+          command: "sourcery.openHub",
+          arguments: [],
+        });
+
+      // Open a webview panel and fill it with a static empty page
+      // The iframe handles loading the actual content
+      const panel = window.createWebviewPanel(
+        "sourceryhub",
+        "Sourcery Hub",
+        ViewColumn.Active,
+          {
+            enableScripts: true,
+          }
+      );
+      panel.webview.html = `<!DOCTYPE html>
+        <head>
+          <style>
+          body,
+          html {
+            width: 100%;
+            height: 100%;
+            margin: 0;
+            padding: 0;
+          }
+          .container {
+            display: flex;
+            width: 100%;
+            height: 100%;
+            flex-direction: column;
+            overflow: hidden;
+          }
+          #theFrame {
+            flex-grow: 1;
+            border: none;
+            margin: 0;
+            padding: 0;
+          }
+        </style>
+      </head>
+      <html lang="en" style="height: 100%; width: 100%; margin: 0; padding: 0">
+        <body style="height: 100%; width: 100%; margin: 0; padding: 0">
+          <div class="container">
+            <iframe
+              id="theFrame"
+              src="http://127.0.0.1:61816/home"
+            ></iframe>
+          </div>
+        </body>
+      </html>`;
+    })
+    );
+
+    // Create the status bar
+    const myStatusBarItem = window.createStatusBarItem(StatusBarAlignment.Left);
+    myStatusBarItem.command = "sourcery.hub.open";
+    myStatusBarItem.text = "Sourcery Hub";
+    myStatusBarItem.tooltip = "Manage your Sourcery account"
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality. -->
**💄 polish** Maybe call this just `Sourcery` and use the tooltip to talk more about `settings` than account

For reference here's tabnine
![image](https://user-images.githubusercontent.com/1440886/197170558-54190fda-825e-41ca-981e-cceee76721c1.png)